src/components/Selected-Movie/Selected-Movie.js

```diff
@@ -11,19 +11,31 @@ const SelectedMovie = ({selectedMovie}) => {
 
         <div className="poster-container">
           <img className="poster-2" src={selectedMovie.poster_path}></img>
-        </div>
-        <div className="movie-info-container">
           <ul className="info-list">
-            <li>movie name: {selectedMovie.title}</li>
             <li>movie rating: {selectedMovie.average_rating.toFixed(2)}</li>
             <li>release date: {selectedMovie.release_date}</li>
+            {selectedMovie.budget ? <li>budget: ${selectedMovie.budget}</li> : '' }
```

Comment:
This is good thinking to do some checking of the data coming in! One thing to think about is what would happen if the `selectedMovie.budget` or `selectedMovie.revenue` is `0`?